setup/download_res.py

```
@@ -229,6 +229,32 @@ def download_deepvariant_model_files(urls: list, outfolder: str):
+        logging.info(f"{keep_file} already exists. Skipping.")
+        return dest_file
+
+    tar_filename = url.split("/")[-1]
+    tar_path = os.path.join(outfolder, tar_filename)
+
+    logging.info(f"Downloading SortMeRNA database from {url}")
+    if not os.path.isfile(tar_path):
+        run_command(["wget", "-c", url, "-P", outfolder])
+
+    logging.info(f"Extracting {keep_file} from archive")
+    run_command(["tar", "-xzf", tar_path, "-C", outfolder, f"--wildcards", f"*/{keep_file}", "--strip-components=1"])
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)